LogCSVWriter.groovy

```diff
@@ -0,0 +1,86 @@
+// -----------------------------------------------------
+// This class is used to output log maps in a CSV format
+// -----------------------------------------------------
+class LogCSVWriter {
+
+    LogCSVWriter(String name = "", Map<String, Integer> map = [:], String path = ""){
+        set_header_name(name)
+        set_map(map)
+        set_output_path(path)
+        output_map_to_file()
+    }
+
+
+    // Getters/Setters
```

Comment:
While I think it is great that this code is very explicit, one of the reasons we write our code in Groovy vs Java is to reduce the boilerplate that we see. It is a code preference among the team to reduce the boilerplate by not explicitly writing getters/setters in Groovy code.